packages/eslint-config-4catalyzer-flow/package.json

```diff
@@ -24,14 +24,11 @@
     "eslint-config-4catalyzer": "^0.8.1"
   },
   "peerDependencies": {
-    "eslint": "^5.15.3",
```

Comment:
I don't think in general we need the explicit `eslint` peer deps here. The other packages we pull in will establish whatever peer dep is required. And anyway this isn't useful if the user doesn't have `eslint` installed.